add_up.rb

```diff
@@ -0,0 +1,16 @@
+#(3) Write a program with a function add_up(i) .  
+#It is to be passed a positive integer, and it will add all the numbers from 1 to that integer and return the sum.  
+#Call the function three times within the program, and each time print out the return value.  
+#Call the program add_up.rb.
+
+def add_up(i)
+    sum = number
+    (1..i).each do |i|
+        sum += i 
```

Comment:
This technically works, but for clarity, it'd be best to define your function parameter different from this #each block parameter. Something like:

(1..i).each do |num| 
   sum += num
end